package.json

```diff
@@ -98,6 +98,8 @@
     "gulp-help": "chmontgomery/gulp-help#gulp4",
     "gulp-if": "2.0.2",
     "gulp-jest": "4.0.2",
+    "gulp-jsonlint": "^1.3.0",
```

Comment:
@erwinmombay Is it alright to add these dependencies? Do we possibly have something to do this already?

Reply:
this should be ok.

@jonathantyng-amp please remove the `^`